docs/resources/cce_cluster.md

```diff
@@ -163,6 +163,26 @@ The following arguments are supported:
 * `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project id of the cce cluster.
   Changing this creates a new cluster.
 
+* `delete_efs` - (Optional, String) Specified whether to delete the associated EFS resources.
```

Comment:
it's better to be described as `Specified whether to delete the associated EFS resources when deleting the CCE cluster.`